releases/release-1.29/release-notes/maps/pr-120492-map.yaml

```diff
@@ -1,5 +1,5 @@
 pr: 120492
 releasenote:
   text: |-
-    Service Controller: update load balancer hosts after node's `ProviderID` is
+    Service Controller: updated load balancer hosts after node's `ProviderID` is
```

Comment:
Service Controller: will now update load balancer hosts after node's `ProviderID` is updated.